### ISSUE TITLE
Channel can't be activated while paying for a spell

### ISSUE THREAD

**Issue by Zerrisx** [magefree/mage#6715]
The special ability granted by Channel can be activated at any time you have priority, but it won't let you do so *during* the cast of a spell.  This is a problem, because you might want to cast a spell while holding priority, and if you activate channel before casting your spell, the life loss could trigger other effects.

Channel ability on an empty stack
![image](https://user-images.githubusercontent.com/59861365/85582283-ed8f4200-b602-11ea-8726-491e8041b1e8.png)

Channel still active, but no way to use it while paying for a spell
![image](https://user-images.githubusercontent.com/59861365/85582383-026bd580-b603-11ea-92cd-59437ba8bd00.png)

**Comment by jeffwadsworth** [comment]
Yeah, the "Special" button has to be coded to stay visible during the "pay cost" part.  Perhaps Jaydi can check this out.  Otherwise, it looks okay to me.

**Comment by jeffwadsworth** [comment]
Just some history.  During Forge's early days, we had a discussion on how to handle effects like this one.  The solution was to put a "card" on the board that had the effect listed, etc.  The "Special" button would just be on that card.
